webrtc.html

```diff
@@ -1891,10 +1891,13 @@
               </dd>
               <dt><code>setConfiguration</code></dt>
               <dd>
-                <p>The <code>setConfiguration</code> method updates the <a>ICE
-                Agent</a> process of gathering local candidates and pinging
-                remote candidates.</p>
-                <p>This call may result in a change to the state of the <a>ICE
+                <p>The <code>setConfiguration</code> method updates the
+                configuration of this <code><a>RTCPeerConnection</a></code>
+                object.  This includes changing configuration of the <a>ICE
+                Agent</a>.</p>
+                <p>Since this call can change parameters relating to gathering
+                local candidates and pinging remote candidates, it may
```

Comment:
To be consistent with ICE terminology, should probably say "performing connectivity checks" instead of "pinging remote candidates".